index.html

```diff
@@ -98,96 +98,55 @@
                 <div class="col-lg-12">
                     <img class="img-responsive" src="assets/img/icons/spongie-mark-dark.png" alt="">
                     <div class="intro-text">
-                        <span class="name">SpongePowered</span>
+                        <span class="name">Sponge</span>
                         <!-- <hr class="star-light"> -->
-                        <span class="skills">A New Minecraft Modding API</span>
+                        <span class="skills">The New Minecraft Plugin API</span>
```

Comment:
That makes it sound too official - also you need to change the title to plugin API